libraries/AP_AHRS/AP_AHRS.h

```diff
@@ -912,6 +926,7 @@ class AP_AHRS {
         float EAS2TAS;
         bool airspeed_ok;
         float airspeed;
+        uint8_t airspeed_estimate_status;
```

Comment:
```suggestion
        ASPD_Status airspeed_estimate_status;
```
Why not use the new enum?

Reply:
Better to leave it as the class as long as possible and case to uint just for the logging.

Reply:
had loads of trouble with this...just reverted to enum and did uint8_t everywhere...will need hand holding to go back to enum class I am afraid